bot/exts/info/information.py

```diff
@@ -145,51 +151,62 @@ async def role_info(self, ctx: Context, *roles: Union[Role, str]) -> None:
     @command(name="server", aliases=["server_info", "guild", "guild_info"])
     async def server_info(self, ctx: Context) -> None:
         """Returns an embed full of server information."""
+        embed = Embed(
+            colour=Colour.blurple(),
+            title="Server Information",
+        )
```

Comment:
Feels like an unnecessary line split again

Reply:
Resolved in 8cdee21.